### ISSUE TITLE
learning semantic role labeling

### ISSUE THREAD

**Issue by omidb** [clearnlp/clearnlp#8]
Hi, 

Is there any documentation on training a semantic role labeler ?

thanks

**Comment by clearnlp** [comment]
Omid,

Sorry there is no document yet :(  I’m really pushing hard to make the next release of ClearNLP which is several months over due.  Good thing is that the semantic role labeler is the only thing that I have to add for this release so hopefully I can get it done soon.  How soon do you need this module working?

best,

Jinho

http://mathcs.emory.edu/~choi/
Assistant professor, Emory University

On Feb 12, 2015, at 4:55 PM, Omid Bakhshandeh <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi,

Is there any documentation on training a semantic role labeler ?

thanks

—
Reply to this email directly or view it on GitHub<https://github.com/clearnlp/clearnlp/issues/8>.


________________________________

This e-mail message (including any attachments) is for the sole use of
the intended recipient(s) and may contain confidential and privileged
information. If the reader of this message is not the intended
recipient, you are hereby notified that any dissemination, distribution
or copying of this message (including any attachments) is strictly
prohibited.

If you have received this message in error, please contact
the sender by reply e-mail message and destroy all copies of the
original message (including attachments).

**Comment by omidb** [comment]
Thanks for reply, I’m working on a paper for ACL and I need to train a semantic role labeler (which is not doing semantic role labeling but the task is exactly the same as semantic role labeling). So I need a SRL that is easy to train with my data and features,


Thanks,


--Omid






Sent from Windows Mail





From: ClearNLP Project
Sent: ‎Friday‎, ‎February‎ ‎13‎, ‎2015 ‎3‎:‎52‎ ‎PM
To: clearnlp/clearnlp
Cc: Omid Bakhshandeh




Omid, 

Sorry there is no document yet :( I’m really pushing hard to make the next release of ClearNLP which is several months over due. Good thing is that the semantic role labeler is the only thing that I have to add for this release so hopefully I can get it done soon. How soon do you need this module working? 

best, 

Jinho 

http://mathcs.emory.edu/~choi/ 
Assistant professor, Emory University 

On Feb 12, 2015, at 4:55 PM, Omid Bakhshandeh <notifications@github.com<mailto:notifications@github.com>> wrote: 


Hi, 

Is there any documentation on training a semantic role labeler ? 

thanks 

— 
Reply to this email directly or view it on GitHub<https://github.com/clearnlp/clearnlp/issues/8>. 


________________________________ 

This e-mail message (including any attachments) is for the sole use of 
the intended recipient(s) and may contain confidential and privileged 
information. If the reader of this message is not the intended 
recipient, you are hereby notified that any dissemination, distribution 
or copying of this message (including any attachments) is strictly 
prohibited. 

If you have received this message in error, please contact 
the sender by reply e-mail message and destroy all copies of the 
original message (including attachments).

**Comment by Masoudn** [comment]
Dear Omid,

I am looking for a similar semantic role labeler which enables me to retrain it with my own data. I was wondering if you have found one? any help would be appreciated

**Comment by omidb** [comment]
@Masoudn I end writing the whole thing from scratch myself. Is your task SRL ?

**Comment by Masoudn** [comment]
Yes, i'm really in the need of applying a semantic role labeler on my own dataset and then retraining semantic role labeler with new data and see if there is any change in the results. Its a long time that i'm looking for it. Could you please help me? really thanks

**Comment by omidb** [comment]
You can use clearNLP if you don't want to write it from scratch, there are also other SRLers out there that you can try, 
or if you can wait, @clearnlp will finish this part soon and you can use that.

**Comment by Masoudn** [comment]
I have been trying ClearNLP for a while but i don't know how to retrain it.
I have also tried Senna but i can't retrain it.
Could you please let me know if there is anyway that i can retrain
ClearNLP? or if there is any other Semantic role labeler that i can
retrain, could you let me know? Thanks



Kind Regards,
Masoud Narouei
masoudnarouei@my.unt.edu

On Tue, Mar 17, 2015 at 10:23 AM, Omid Bakhshandeh <notifications@github.com

**Comment by clearnlp** [comment]
Masoud,

The ClearNLP version 3.0.0 is about to be released and it gives a much better way of training.  Unfortunately, the semantic role labeler will be integrated towards the end of this semester (sometime in April) if you can wait.  Please let me know if you have any question.  Thanks!

best,

Jinho

http://mathcs.emory.edu/~choi/
Assistant professor, Emory University

On Mar 17, 2015, at 11:27 AM, Masoudn <notifications@github.com<mailto:notifications@github.com>> wrote:

I have been trying ClearNLP for a while but i don't know how to retrain it.
I have also tried Senna but i can't retrain it.
Could you please let me know if there is anyway that i can retrain
ClearNLP? or if there is any other Semantic role labeler that i can
retrain, could you let me know? Thanks



Kind Regards,
Masoud Narouei
masoudnarouei@my.unt.edu<mailto:masoudnarouei@my.unt.edu>

On Tue, Mar 17, 2015 at 10:23 AM, Omid Bakhshandeh <notifications@github.com<mailto:notifications@github.com>
> wrote:

> You can use clearNLP if you don't want to write it from scratch, there are
> also other SRLers out there that you can try,
> or if you can wait, @clearnlp <https://github.com/clearnlp> will finish
> this part soon and you can use that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clearnlp/clearnlp/issues/8#issuecomment-82409786>.
>

—
Reply to this email directly or view it on GitHub<https://github.com/clearnlp/clearnlp/issues/8#issuecomment-82412683>.


________________________________

This e-mail message (including any attachments) is for the sole use of
the intended recipient(s) and may contain confidential and privileged
information. If the reader of this message is not the intended
recipient, you are hereby notified that any dissemination, distribution
or copying of this message (including any attachments) is strictly
prohibited.

If you have received this message in error, please contact
the sender by reply e-mail message and destroy all copies of the
original message (including attachments).
